### ISSUE TITLE
[Bug]: Cannot find version 8.0.mysql_aurora.3.02.0 for aurora-mysql

### ISSUE THREAD

**Issue by ruzickap** [aws/copilot-cli#5812]
### Description:

I'm getting error using `copilot storage init`: `Cannot find version 8.0.mysql_aurora.3.02.0 for aurora-mysql`
 
### Details:

I would like to use copilot with Aurora - so I tried the following command:

```bash
copilot storage init --name wordpress-cluster --lifecycle=workload \
  --storage-type Aurora --engine MySQL --initial-db "wordpress"
``` 

After running `copilot svc deploy` I got:

```
Found only one service, defaulting to: wordpress
Only found one option, defaulting to: dev
✔ Proposing infrastructure changes for stack wordpress-dev-wordpress
- Creating the infrastructure for stack wordpress-dev-wordpress                                       [rollback complete]   [377.5s]
  The following resource(s) failed to create: [AddonsStack]. Rollback re
  quested by user.
  - An Addons CloudFormation Stack for your additional AWS resources                                  [delete in progress]  [377.9s]
    The following resource(s) failed to create: [wordpressclusterDBCluster
    ]. Rollback requested by user.
    - A Secrets Manager secret to store your DB credentials                                           [delete complete]     [10.2s]
    - A DB parameter group for engine configuration values                                            [delete complete]     [0.0s]
    - A security group for your Aurora Serverless v2 cluster wordpresscluster                         [delete complete]     [1.3s]
    - The wordpresscluster Aurora Serverless v2 database cluster                                      [delete complete]     [0.0s]
      Resource handler returned message: "Cannot find version 8.0.mysql_auro
      ra.3.02.0 for aurora-mysql (Service: Rds, Status Code: 400, Request ID
      : d2a3b5e3-c96b-49f1-b330-2ef020e01c94)" (RequestToken: 7d57531f-3bae-
      c1fb-48e4-947f089f7bae, HandlerErrorCode: InvalidRequest)
```

It looks like CloudFormation template generated by the `copilot storage init` is not valid anymore.

The CF template contains:

```yaml
...
  wordpressclusterDBCluster:
    Metadata:
      'aws:copilot:description': 'The wordpresscluster Aurora Serverless v2 database cluster'
    Type: 'AWS::RDS::DBCluster'
    Properties:
      MasterUsername:
        !Join [ "",  [ '{{resolve:secretsmanager:', !Ref wordpressclusterAuroraSecret, ":SecretString:username}}" ]]
      MasterUserPassword:
        !Join [ "",  [ '{{resolve:secretsmanager:', !Ref wordpressclusterAuroraSecret, ":SecretString:password}}" ]]
      DatabaseName: !Ref wordpressclusterDBName
      Engine: 'aurora-mysql'
      EngineVersion: '8.0.mysql_aurora.3.02.0'
      DBClusterParameterGroupName: !Ref wordpressclusterDBClusterParameterGroup
      DBSubnetGroupName: !Ref wordpressclusterDBSubnetGroup
...
```

My copilot version

```bash
❯ copilot --version
copilot version: v1.33.3
```

Thank you for looking at it...

**Comment by hkford** [comment]
Thank you for reporting the issue. It appears that the root cause of the problem is that the AWS Copilot CLI is [currently hardcoded to use MySQL version 3.02.0 when creating Aurora Serverless v2 clusters](https://github.com/aws/copilot-cli/blob/v1.33.3/internal/pkg/template/templates/addons/aurora/serverlessv2.yml#L123). [Depending on the AWS region you are using](https://docs.aws.amazon.com/AmazonRDS/latest/AuroraUserGuide/Concepts.Aurora_Fea_Regions_DB-eng.Feature.ServerlessV2.html#Concepts.Aurora_Fea_Regions_DB-eng.Feature.ServerlessV2.amy), a higher version of MySQL may be required.

**Comment by ruzickap** [comment]
Thank you.

I'm using `us-east-1` and `eu-central-1`. 

Anyway - based on the output of the script below - it seems like version `8.0.mysql_aurora.3.02.0` is not supported "anywhere".

```bash
for REGION in `aws ec2 describe-regions --region us-east-1 --output text | cut -f4`; do
  echo "*** ${REGION}"
  aws rds describe-db-engine-versions --region "${REGION}" --engine aurora-mysql --query "DBEngineVersions[].EngineVersion" | grep '8.0.mysql_aurora.3.02.0'
done
```

Let me create PR to fix it...

**Comment by dannyrandall** [comment]
The fix is now released in v1.33.4: https://github.com/aws/copilot-cli/releases/tag/v1.33.4! 🎉  Thanks @ruzickap for the PR!
